config/init_server_creds.go

```diff
@@ -147,6 +261,12 @@ func GenerateCert() error {
 		return err
 	}
 
+	caPrivateKey, err := LoadPrivateKey(viper.GetString("TLSCAKey"))
```

Comment:
I took a slightly different tack.

If we don't have a valid CA key file, do a self-signed certificate instead of a CA-based one.

This could be improved in 7.2.0 to always generate a CA (the basic issue you're hitting above is the fact we're using the system CAs)... but I think I'm OK with splitting out that improvement to get this PR across the finish line.